scripts/ci/distcheck.sh

```diff
@@ -1,25 +1,17 @@
 #!/bin/bash
 set -e
 
-# CC LD AR CXX has to be predifubed
-#
+if [ "${CC#clang}" != "${CC}" ] ; then
+	export CXX="clang++"
```

Comment:
interesting will CXX=`echo $CC | sed 's/clang/clang++/'`  work here. If we provide clang-3.8  for example it will use correct version for C++.

Reply:
web hides execution quotes.

Reply:
Ugh. Then `${CC/clang/clang++}` would be easier.